partialWaveFit/fitResult.h

```diff
@@ -132,18 +133,22 @@ namespace rpwa {
 		          const bool                                hasHessian);             // indicates whether Hessian matrix has been calculated successfully
 		void fill(const fitResult& result);
 
-		unsigned int        nmbEvents         () const { return _nmbEvents;         }  ///< returns number of events in bin
-		unsigned int        normNmbEvents     () const { return _normNmbEvents;     }  ///< returns number of events to normalize to
-		double              massBinCenter     () const { return _massBinCenter;     }  ///< returns center value of mass bin
-		double              logLikelihood     () const { return _logLikelihood;     }  ///< returns log(likelihood) at maximum
-		double              evidence          () const;                                ///< returns the model evidence (OccamFactorMethod)
-		std::vector<double> evidenceComponents() const;                                ///< returns a vector { maxLogL, ln(sqrt((2\pi)^m*cov)), -ln(V_A^k), \sum(S_\alpha) }, i.e. evidence = sum(evidenceComponents[i]) for i in {1,2,3,4}
-		unsigned int        rank              () const { return _rank;              }  ///< returns rank of fit
-		bool                covMatrixValid    () const { return _covMatrixValid;    }
-		bool                converged         () const { return _converged;         }  ///< returns whether fit has converged (according to minimizer)
-		bool                hasHessian        () const { return _hasHessian;        }  ///< returns whether Hessian matrix has been calculated successfully
-		unsigned int        nmbWaves          () const { return _waveNames.size();  }  ///< returns number of waves in fit
-		unsigned int        nmbProdAmps       () const { return _prodAmps.size();   }  ///< returns number of production amplitudes
+		void                  setBinningMap     (const binningMapType& binningMap)   { _binningMap = binningMap; } ///< set binning map
+
+		unsigned int          nmbEvents         () const { return _nmbEvents;               }  ///< returns number of events in bin
+		unsigned int          normNmbEvents     () const { return _normNmbEvents;           }  ///< returns number of events to normalize to
+		double                massBinCenter     () const { return multibinCenter()["mass"]; }  ///< returns center value of mass bin
```

Comment:
I am not sure if we should not entirely remove this function. This would probably require quite some changes in code using the fit result, but I am not sure the binning maps are even required to contain a `mass` variable by the file manager.